doc/suiterc.tex

```diff
@@ -1265,6 +1265,46 @@ \subsection{[runtime]}
 \item {\em default:} (none)
 \end{myitemize}
 
+\subparagraph[retrieve job log]{[runtime] $\rightarrow$ [[\_\_NAME\_\_]] $\rightarrow$ [[[remote]]] $\rightarrow$ retrieve job log}
+
+Remote task job logs are saved to the suite run directory on the task host, not
+on the suite host. If you want the job logs pulled back to the suite host
+automatically, you can set this setting to \lstinline=True=. The suite will
+then attempt to \lstinline=rsync= the job logs once from the remote host each
+time a task job completes. E.g. if the job file is
+\lstinline=~/cylc-run/tut.oneoff.remote/log/job/1/hello/01/job=, anything under
+\lstinline=~/cylc-run/tut.oneoff.remote/log/job/1/hello/01/= will be retrieved.
+
+\begin{myitemize}
+\item {\em type:} boolean
+\item {\em default:} False
+\end{myitemize}
+
+\subparagraph[retrieve job log max size]{[runtime] $\rightarrow$ [[\_\_NAME\_\_]] $\rightarrow$ [[[remote]]] $\rightarrow$ retrieve job log max size}
+
+If the disk space of the suite host is limited, you may want to restrict the
+maximum sizes of the job log files to retrieve by setting a maximum size. The
```

Comment:
"restrict the maximum size ... by setting a maximum size" sounds a bit circular.  Maybe "... with this item."